default.nix

```diff
@@ -109,6 +109,7 @@ let overrideCabal = pkg: f: if pkg == null then null else lib.overrideCabal pkg
         reflex-dom = addReflexOptimizerFlag reflexDom.reflex-dom;
         reflex-dom-core = addReflexOptimizerFlag reflexDom.reflex-dom-core;
         reflex-todomvc = self.callPackage ./reflex-todomvc {};
+        cabal-macosx = self.callPackage ../cabal-macosx {};
```

Comment:
This doesn't look right - we can't refer to things in ..; however, we can set up cabal-macos like our other "submodule"-style projects and then do `self.callPackage ./cabal-macosx {}`.